android/build.gradle

```diff
@@ -2,21 +2,22 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "29.0.2"
+        buildToolsVersion = "29.0.3"
         minSdkVersion = 21
         compileSdkVersion = 29
         targetSdkVersion = 29
         supportLibVersion = "28.0.0"
         googlePlayServicesVersion = "16.+"
         firebaseVersion = "17.3.4"
         kotlinVersion = '1.4.10'
+        ndkVersion = "20.1.5948944"
     }
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath("com.android.tools.build:gradle:3.5.3")
+        classpath("com.android.tools.build:gradle:4.1.0")
```

Comment:
need changes in bitrise ?

Reply:
Bitrise is ok, I did all changed it required